lib/matplotlib/axes/_axes.py

```diff
@@ -1452,19 +1453,20 @@ def plot(self, *args, scalex=True, scaley=True, **kwargs):
         Alternatively, you can also change the style cycle using the
         'axes.prop_cycle' rcParam.
 
+
         Parameters
         ----------
         x, y : array-like or scalar
             The horizontal / vertical coordinates of the data points.
-            *x* values are optional. If not given, they default to
-            ``[0, ..., N-1]``.
+            *x* values are optional and default to `range(len(y))`.
+
+            Commonly, these parameters are 1D arrays.
 
-            Commonly, these parameters are arrays of length N. However,
-            scalars are supported as well (equivalent to an array with
-            constant value).
+            They can also be scalars, or two-dimensional (in that case, the
+            columns represent separate data sets).
 
-            The parameters can also be 2-dimensional. Then, the columns
-            represent separate data sets.
+            These arguments cannot be passed as keywords (as indicated by the
```

Comment:
That helps, but fmt can't be passed ask warg either according to the signature?

Reply:
added a note there too